packages/react-components/src/molecules/ListControls.tsx

```diff
@@ -1,43 +1,24 @@
-import { Link } from 'react-router-dom';
 import { css } from '@emotion/react';
-
-import { charcoal, fern, lead } from '../colors';
-import { perRem, lineHeight } from '../pixels';
-import { cardViewIcon, listViewIcon } from '../icons';
+import { dropdownChevronIcon } from '../icons';
+import DropdownButton from './DropdownButton';
+import { rem, tabletScreen } from '../pixels';
 
 const containerStyles = css({
-  display: 'grid',
-  gridAutoFlow: 'column',
-  columnGap: '24px',
-  width: 'fit-content',
-  margin: `${12 / perRem}em 0`,
-});
-
-const linkStyles = css({
-  color: charcoal.rgb,
-  textDecoration: 'none',
-  outline: 'none',
-});
-
-const textStyles = css({
-  margin: 0,
   display: 'flex',
-  fontSize: `${18 / perRem}em`,
-  svg: {
-    stroke: lead.rgb,
+  alignItems: 'center',
+  gap: rem(15),
+  'span + svg': {
+    width: '11px',
+  },
+  button: {
+    padding: `${rem(8)} ${rem(24)}`,
```

Comment:
I think this padding `rem(24)` is not matching what we see in figma. It seems there is an extra space on the left side.

![Screenshot 2024-01-05 at 13 14 32](https://github.com/yldio/asap-hub/assets/16595804/fb778b38-24a1-42c0-b27c-b684a966551d)